modules/installation-launching-installer.adoc

```diff
@@ -429,6 +429,14 @@ ifdef::vsphere[]
 .. Specify the user name and password for the vCenter account that has the required permissions to create the cluster.
 +
 The installation program connects to your vCenter instance.
++
+[IMPORTANT]
+====
+Some VMware vCenter Single Sign-On (SSO) environments with Active Directory (AD) integration might primarily require you to use the legacy login method; that is, the `<domain>\` construct.
+
+To ensure vCenter account permission checks complete properly, consider using the User Principal Name (UPN) login method, such as `<username>@<fully_qualified_domainname>`.
```

Comment:
```suggestion
To ensure that vCenter account permission checks complete properly, consider using the User Principal Name (UPN) login method, such as `<username>@<fully_qualified_domainname>`.
```

It might also be worth considering swapping , such as / , for example,